string name
string frame_id

float32 x_center
float32 y_center
float32 height
float32 width
